scripts/install_submodules.sh

```diff
@@ -78,7 +41,8 @@ stage "Clone submodules"
 
 cd "${PLATFORM_PATH}" && git submodule update --init --recursive
 
-clone_project "${SC_MACHINE_REPO}" "${SC_MACHINE_PATH}" "${SC_MACHINE_BRANCH}" "${SC_MACHINE_COMMIT}"
-clone_project "${SC_WEB_REPO}" "${SC_WEB_PATH}" "${SC_WEB_BRANCH}" "${SC_WEB_COMMIT}"
+"${PLATFORM_PATH}/scripts/install_sc_machine_submodule.sh" "$1"
+"${PLATFORM_PATH}/scripts/install_sc_web_submodule.sh" "$1"
+"${PLATFORM_PATH}/scripts/install_ims_kb_submodule.sh" "$1"
```

Comment:
```suggestion
"${PLATFORM_PATH}/scripts/install_sc_machine_submodule.sh" "$@"
"${PLATFORM_PATH}/scripts/install_sc_web_submodule.sh" "$@"
"${PLATFORM_PATH}/scripts/install_ims_kb_submodule.sh" "$@"
```

Reply:
why? "$0" is a absolute path of executing script